repose-aggregator/components/filters/header-user-filter/src/test/scala/org/openrepose/filters/headeruser/config/HeaderUserSchemaTest.scala

```diff
@@ -20,21 +20,20 @@
 
 package org.openrepose.filters.headeruser.config
```

Comment:
We should standardize on where we are putting this test. Just over half are in the `org.openrepose.filters.filtername` package (16) and others are a layer deeper in the `org.openrepose.filters.filtername.config` package (14).

Reply:
That's fair. Sticking it in the `config` package means that I don't have to import the `ObjectFactory` in most if not all cases, so I would suggest putting them there.

I'll move them.